Zukini/Zukini.csproj

```diff
@@ -36,6 +36,9 @@
     <Reference Include="Newtonsoft.Json, Version=10.0.0.0, Culture=neutral, PublicKeyToken=30ad4fe6b2a6aeed, processorArchitecture=MSIL">
       <HintPath>..\packages\Newtonsoft.Json.10.0.3\lib\net45\Newtonsoft.Json.dll</HintPath>
     </Reference>
+    <Reference Include="nunit.framework, Version=3.9.0.0, Culture=neutral, PublicKeyToken=2638cd05610744eb, processorArchitecture=MSIL">
+      <HintPath>..\packages\NUnit.3.9.0\lib\net45\nunit.framework.dll</HintPath>
+    </Reference>
```

Comment:
Why do we need the NUnit.Framework reference in the core Zukini.csproj now?

Reply:
I'm using NUnit.Framework.AssertionException from NUnit for negative situations - for example when element not found etc. Do you think we should go with some core .Net exception for this? 

Which Error / Exception do you recommend to use?

The point was to prevent users from doing:
```
var abc = someList.FirstOrDefault();
Assert.IsNotNull(abc)
```

Reply:
I would really rather prefer we don't bind ourselves to NUnit at the Zukini level. It is fine for the Example features and such. Can you simply write your own exception and replicate the behavior?

Reply:
Agree. Will do.